### ISSUE TITLE
Consolidate `GeoTessModel` methods related to `.getWeights` 

### ISSUE THREAD

**Issue by jkmacc-LANL** [LANL-Seismoacoustics/PyGeoTess#15]
Right now, they're in different methods with different calling signatures, but they could be consolidated to remain in-step with the C++ `.getWeights` API.

**Comment by rad-lanl** [comment]
I agree, I'm not quite sure how this might look but I don't mind changing any of my codes to fall in line with this!
